docs/src/modules/components/Demo.js

```diff
@@ -210,7 +210,7 @@ const DemoRootMaterial = styled('div', {
   display: 'flex',
   justifyContent: 'center',
   [theme.breakpoints.up('sm')]: {
-    borderRadius: '12px 12px 0 0',
+    borderRadius: hiddenToolbar ? 12 : '12px 12px 0 0',
```

Comment:
Fix border-radius at the footer of https://mui.com/material-ui/react-grid/#interactive